content/docs/guides/continuous-delivery/_index.md

```diff
@@ -1,7 +1,6 @@
 ---
-title: Continuous Delivery
-meta_desc: This page provides an overview of how to integrate Pulumi with your current CI/CD
-           processes to review, validate, and test cloud resources.
+title: Continuous Delivery for Cloud Resources
+meta_desc: Pulumi's approach to infrastructure as code is great for continuous delivery, because it uses source code to model cloud resources. Pulumi can easily integrate into any CI/CD system.
```

Comment:
```suggestion
meta_desc: Pulumi's approach to infrastructure as code is great for CI/CD because it uses source code to model cloud resources and integrates into any CI/CD system.
```